templates/ci/python-django.yml

```diff
@@ -0,0 +1,47 @@
+# Python Django
+# Test a Django app on multiple versions of Python.
+# https://aka.ms/yaml
+
+phases:
+
+# =================
+# Test a Django app
+# =================
+- phase: 'DjangoApp'
+  variables:
+    # relative path to your manage.py
+    projectPath: mysite/
+
+  queue:
+    name: 'Hosted Linux Preview'
+    parallel: 1
+    matrix:
+      Python35:
+        PYTHON_VERSION: '3.5'
+      Python36:
+        PYTHON_VERSION: '3.6'
+      Python37-dev:
+        PYTHON_VERSION: '>= 3.7.0a'
+  steps:
+
+  - task: UsePythonVersion@0
+    inputs:
+      versionSpec: '$(PYTHON_VERSION)'
+      architecture: 'x64'
+
+  - script: python -m pip install --upgrade pip setuptools wheel
+    displayName: 'Install dependencies'
+
+  - script: python -m pip install -r requirements.txt
```

Comment:
Just curious, why do you use `python -m pip` instead of just `pip`?

Reply:
I tried it using bare `pip install` and it was picking up some system-wide pip 8.x that didn't have setuptools installed. This in turn caused Django to fail to install.

I'm open to changing it if I've misunderstood something.

Reply:
Oh right, that was a bug on Linux agents with version 133 of Use Python Version.  It's fixed in 134.

Reply:
Oh, cool. Well, I need the hosted pool to pick up that change and then I'll change this back.

Reply:
FWIW, when upgrading pip on Windows it is better to do `python -m pip`, as it avoids trouble when it updates the `pip.exe` script.

Reply:
Good to know! I'll leave it this way -- even though the task runs on Linux by default, better to have the xplat working.